openpype/hosts/blender/plugins/publish/pause_sync_server.py

```diff
@@ -0,0 +1,17 @@
+import pyblish
```

Comment:
This plugin `pause_sync_server` and `unpause_sync_server` does not belong to this PR. If they should be added to core, they should be in SyncServer, but that should happen in different PR.

Reply:
Both are importing plugins from different places, that is not ok.